ETC.tsv

```diff
@@ -916,7 +916,7 @@ ETC_20150317_000915	Woodfung
 ETC_20150317_000916	The Woodfung is closely related to witchcraft.
 ETC_20150317_000917	{memo X}Red Infrorocktor
 ETC_20150317_000918	They exhibit a unique hull that is a mix between stone and iron.
-ETC_20150317_000919	Contaminated Vikaras Mage
+ETC_20150317_000919	Corrupt Vikaras Mage
```

Comment:
I think he's right though.  Aren't these the passed out mages in the mausoleum?  They've been "Corrupted", and are in that state, rather than mages that were once good, but are now Corrupt and fighting against you?